docs/ldml/tr35-keyboards.md

```diff
@@ -981,6 +987,8 @@ _Attribute:_ `base`
 _Attribute:_ `path` (required)
 
 > If `base` is `cldr`, then the `path` must start with a CLDR version (such as `techpreview`) representing the CLDR version to pull imports from. The imports are located in the `keyboard/import` subdirectory of the CLDR source repository.
+> Implementations are not required to have any or all all CLDR versions available to them. An implementation may reject a keyboard file which references an unavailable CLDR version.
```

Comment:
"reject" is imprecise phrasing. Must not support? Must not allow the user to select? 
What does "unavailable" mean? Unavailable to the implementation? Non-existent?

Reply:
> Implementations are not required to have any or all CLDR versions available to them.

We could just leave it at that.

Reply:
> "reject" is imprecise phrasing. Must not support? Must not allow the user to select? 
> 
> What does "unavailable" mean? Unavailable to the implementation? Non-existent?
> 
> 
> 
> 
> 
> 
> 
> 

Jan's idea is to just drop the sentence.

Reply:
```suggestion
> Implementations are not required to have any or all all CLDR versions available to them.
```